kr/kr_pgp.go

```diff
@@ -322,3 +323,89 @@ func uninstallCodesigning() {
 	exec.Command("git", "config", "--global", "--unset", "commit.gpgSign").Run()
 	exec.Command("git", "config", "--global", "--unset", "tag.forceSignAnnotated").Run()
 }
+
+func pgpSignCommand(c *cli.Context) (err error) {
+	message := c.String("message")
+	if message == "" {
+		stat, err := os.Stdin.Stat()
+		if err != nil {
+			PrintFatal(os.Stderr, "error reading stdin: "+err.Error())
+			return err
+		}
+		if (stat.Mode() & os.ModeCharDevice) == 0 {
+			_, err = fmt.Scanf("%s", &message)
```

Comment:
This actually only reads the first word in the input -- you can read all of the bytes of stdin like this: https://github.com/kryptco/kr/blob/master/krgpg/krgpg.go#L122 .